src/content/docs/vulnerability-management/overview.mdx

```diff
@@ -26,49 +26,35 @@ Vulnerability Management provides a birds-eye view of all your software's vulner
 * Calculate the vulnerability surface area of your software
 * Triage, prioritize, and remediate vulnerabilities with steps provided
 
-[Visit one.newrelic.com](https://one.newrelic.com/vulnerability-management) to get started today.
+## View security insights for developers, security teams, and architects
```

Comment:
This is redundant because it's mentioned below in the "Monitor your vulnerabilities now" section